doc/nrf/protocols/lte/psm.rst

```diff
@@ -0,0 +1,272 @@
+
+Before sending data, the application can request the modem to estimate energy efficiency of the cell that it connects to.
+The :ref:`lte_lc_readme` can be used to query this information.
+The following code shows how to query this information:
+
+.. code-block:: c
+
+   struct lte_lc_conn_eval_params params = {0};
+   int err = lte_lc_conn_eval_params_get(&params);
+   if (err) {
+      printk("lte_lc_conn_eval_params_get, error: %d\n", err);
+      return;
+    }
+
+The following code block yields a high-level measure that can be used to delay network operations:
+
+.. code-block:: c
+
+   if (params.energy_estimate < LTE_LC_ENERGY_CONSUMPTION_NORMAL) {
+      /* start delay timer */
+    } else {
+      /* do network operation immediately */
+    }
+
+Synchronizing application with modem sleep
+==========================================
+
+Depending on network configuration, the modem wakes up at certain intervals.
+For example, it will have to do a Tracking Area Update (TAU) periodically to avoid reconnecting to the network.
+It is possible to subscribe to :c:enum:`LTE_LC_EVT_TAU_PRE_WARNING` or :c:enum:`LTE_LC_EVT_MODEM_SLEEP_EXIT_PRE_WARNING` events and then send application data whenever the modem wakes up.
+Use the pre-warning notifications to avoid having to wake up twice.
+See the :ref:`lte_lc_readme` documentation for more information.
+
+Low battery behavior
+====================
+
+You can configure low battery level warnings using the ``AT%XVBATLOWLVL`` command and subscribe using the ``AT%XVBATLVL`` command.
+There is also a power-off feature that notifies the application when the modem has shut down because of low voltage.
+It can be configured using the ``AT%XPOFWARN`` command.
+
+Release Assistance Indication (RAI)
+===================================
+
+If you have low-level control over the protocol your IoT device uses, you might know when you should not expect more data.
+The IoT devices informs the network that they do not expect any more data to be sent or received after their current transmission using RAI.
+See :ref:`release_assistance_indication` for more information.
+
+There are two types of RAI:
+
+* Control Plane RAI (CP-RAI) - Used for control plane data in NB-IoT.
+* Access Stratum RAI (AS-RAI) - Used for both control plane and user plane data in LTE-M and NB-IoT.
+
+The section below explins the Access Stratum (AS-RAI) in detail.
+
+.. _cellular_as_rai:
+
+Access Stratum (AS-RAI)
+-----------------------
+
+The UE in RRC connected mode will always transfer data and then wait for an RRC inactivity timer period before switching to RRC idle mode, regardless of the power-saving strategy used.
+
+The RRC inactivity timer period is set by the network and cannot be negotiated by the UE.
+Therefore, when waiting to transit to RRC idle mode, a large timer value may unnecessarily increase power usage.
+In RRC Connected mode, If the UE keeps the radio on for an extra 60 seconds each time.
```

Comment:
Could this sentence be removed? It talks about 60 seconds, while the next sentence talks about 5-60 seconds, so at least that should be aligned. But, I think the whole sentence could be removed, the message should be clear without as well.